pkg/gui/presentation/icons/file_icons.go

```diff
@@ -326,7 +328,12 @@ func patchFileIconsForNerdFontsV2() {
 	extIconMap[".vue"] = IconProperties{Icon: "\ufd42", Color: 113}     // ﵂
 }
 
-func IconForFile(name string, isSubmodule bool, isLinkedWorktree bool, isDirectory bool) IconProperties {
```

Comment:
Is this something the linter changed or?

Reply:
I have fixed this now